### PR TITLE
add file diary

### PR DESCRIPTION
**Pull request by XiangRn**

### CODE REVIEW

File: diary.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function diary(){
```
Comment (XiangBuss) [inline diary.js:1]: add diary name on this place

Comment (XiangRn) [reply diary.js:1]: Thanks bro

File: diary.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+function diary(name){
+	this.name=name;
+  this.stomach=[];
+}
+module.exports=diary;
+"Ok?"
```
Comment (XiangRn) [inline diary.js:6]: Why do u add this line?

Comment (XiangBuss) [reply diary.js:6]: I decide to ask you

Comment (XiangRn) [reply diary.js:6]: oh, everything is ok, You need delete this line, Thanks bro

Comment (XiangBuss) [reply diary.js:6]: uhmmm

**Pull request merged**